src/widgets/splits/SplitHeader.cpp

```diff
@@ -569,12 +569,25 @@ void SplitHeader::updateChannelText()
         if (streamStatus->live)
         {
             this->isLive_ = true;
-            if (!this->lastThumbnail_.isValid() ||
-                this->lastThumbnail_.elapsed() > 5 * 60 * 1000)
+            QString url = "https://static-cdn.jtvnw.net/"
+                          "previews-ttv/live_user_" +
+                          channel->getName().toLower();
+            switch (getSettings()->thumbnailSizeStream.getValue())
+            {
+                case 1:
+                    url.append("-80x45.jpg");
+                    break;
+                case 2:
+                    url.append("-160x90.jpg");
+                    break;
+                case 3:
+                    url.append("-360x180.jpg");
+                    break;
```

Comment:
I would like to see a `default` case where url is set to an empty string or something which can then be checked in the `if`-case below